docker-for-aws/faqs.md

```diff
@@ -150,3 +150,10 @@ $ sudo ping 10.0.0.4
 ```
 
 > **Note**: Access to Docker for AWS and Azure happens through a shell container that itself runs on Docker.
+
+## How do I uninstall Docker for AWS?
```

Comment:
I just dislike having questions as headers. Why wouldn't it just be "Uninstall Docker for AWS"? I feel like maybe I've given this feedback before, so if so, feel free to keep ignoring it. ;)

Reply:
I'm not going to change header content, too much risk that links to this could be broken